kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/SectionEditor.vue

```diff
@@ -16,59 +15,57 @@
       :invalidText="sectionTitleInvalidText"
       :maxlength="100"
     />
-
     <KTextbox
       v-model="description"
       :label="optionalDescriptionLabel$()"
       :maxlength="400"
       :textArea="true"
       class="description-ktextbox-style"
     />
-
     <hr :style="dividerStyle" >
-
     <div>
       <h5 class="section-settings-heading">
         {{ questionOrder$() }}
       </h5>
       <KGrid>
-        <KGridItem
-          :layout12="{ span: 6 }"
-          :layout8="{ span: 4 }"
-          :layout4="{ span: 2 }"
-        >
-          <KRadioButton
-            v-model="learners_see_fixed_order"
-            :label="randomizedLabel$()"
-            :buttonValue="false"
-            :description="randomizedOptionDescription$()"
-          />
-        </KGridItem>
-        <KGridItem
-          :layout12="{ span: 6 }"
-          :layout8="{ span: 4 }"
-          :layout4="{ span: 2 }"
-        >
-          <KRadioButton
-            v-model="learners_see_fixed_order"
-            :label="fixedLabel$()"
-            :buttonValue="true"
-            :description="fixedOptionDescription$()"
-          />
-        </KGridItem>
+        <KRadioButtonGroup>
+          <KGridItem
+            :layout12="{ span: 6 }"
+            :layout8="{ span: 4 }"
+            :layout4="{ span: 2 }"
+          >
+            <KRadioButton
+              v-model="learners_see_fixed_order"
+              :label="randomizedLabel$()"
+              :buttonValue="false"
+              :description="randomizedOptionDescription$()"
+            />
+          </KGridItem>
+        </KRadioButtonGroup>
```

Comment:
Two things - do we need to have `KRadioButtonGroup` for a single radio button?

Also, as I mentioned in another part of the PR, the `KRadioButtonGroup` is wrapped around some structural components (here, `KGrid*`) - but it should only wrap around the smallest block of code that it can to avoid possible conflicts.

Reply:
@nucleogenesis I understand you concern about wrapping structural components and single radio buttons but I’ve followed the approach from a referenced merged PR #12325 mentioned in original issue #12596  where `KRadioButtonGroup` wrapped single radio buttons and some components like `KGrid` with nested `KRadioButtons`. 

But I am open to further adjustments if this doesn't meet the intended requirements or if it conflicts with any specific design patterns or functionality. Let me know if there’s a preferred approach, or if we should explore additional conditions to refine the wrapping process further.

Reply:
Hi

@nucleogenesis, regarding

> Also, as I mentioned in another part of the PR, the KRadioButtonGroup is wrapped around some structural components (here, KGrid*) - but it should only wrap around the smallest block of code that it can to avoid possible conflicts.

Wrapping the smallest block of code would require using more `KRadioButtonGroup`'s whereas we rather need to have exactly one `KRadioButtonGroup` wrapping a group of logically related radio buttons (including nested ones). Otherwise, keyboard navigation would not work as expected. There's an example illustrating this on grids in the "With grid layout" section of https://develop--kolibri-design-system.netlify.app/kradiobutton/. If you have ideas on how to better document this, please let me know.

Reply:
Looking into the documentation now myself, I think we should at least mention something like

> to have exactly one KRadioButtonGroup wrapping a group of logically related radio buttons 

in the Guidelines. It's not explicitly said at this moment.

Reply:
I am not reviewing other places, just an example for this particular place, here I believe we'd need:

```
<KGrid>
  <KRadioButtonGroup>
      <KGridItem
        :layout12="{ span: 6 }"
        :layout8="{ span: 4 }"
        :layout4="{ span: 2 }"
      >
        <KRadioButton
          v-model="learners_see_fixed_order"
          :label="randomizedLabel$()"
          :buttonValue="false"
          :description="randomizedOptionDescription$()"
        />
      </KGridItem>
      <KGridItem
        :layout12="{ span: 6 }"
        :layout8="{ span: 4 }"
        :layout4="{ span: 2 }"
      >
        <KRadioButton
          v-model="learners_see_fixed_order"
          :label="fixedLabel$()"
          :buttonValue="true"
          :description="fixedOptionDescription$()"
        />
      </KGridItem>
  <KRadioButtonGroup>
</KGrid>
```

rather than two `KRadioButtonGroup`s.

It'd be always best to validate keyboard navigation in Firefox as final criteria https://github.com/learningequality/kolibri/issues/10491 so that's what I'd recommend @iamshobhraj. Also adding @radinamatic for final QA as soon as all changes requested so far are resolved.

Reply:
@MisRob can we have  `KRadioButtonGroup` for a single radio button?

Reply:
> can we have KRadioButtonGroup for a single radio button?

I assume that either having or not having it wouldn't really have impact on user experience since https://github.com/learningequality/kolibri/issues/10491 really concerns only groups of more buttons - that's where we need the keyboard navigation with arrows to work. Right, @radinamatic? I was actually surprised to hear we have a single radio button and I wonder what's the use-case, would you post a screenshot of that place @iamshobhraj, for context?

Reply:
I agree, single radio button makes no sense (and group markup around it even less) 👍🏽

Reply:
Alright thank you @radinamatic. So @iamshobhraj, please do not wrap a single `KRadioButton` in `KRadioButtonGroup` (however, if you can, it'd be still useful to see where you observed this place, and we will then assess if we need to use a different element - that'd be done in the scope of another issue though)

Reply:
@MisRob certain component files, like [LanguagesMenuItem](https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/media_player/assets/src/views/MediaPlayerLanguages/LanguagesMenuItem.vue) contains a single `KRadioButton` because the component renders itself multiple times elsewhere. Should these have a `KRadioButtonGroup` wrapper?

Reply:
@iamshobhraj

The keyboard navigation as described in https://github.com/learningequality/kolibri/issues/10491 won't work if you did something like

```
<KRadioButtonGroup>
  <KRadioButton>
</KRadioButtonGroup>

<KRadioButtonGroup>
  <KRadioButton>
</KRadioButtonGroup>
```
 
The deciding factor is resulting markup and user experience so for components like this you'd need to do search and understand the places where they're being used from.

What I can say from the code itself, I'd expect to see something like

```
<KRadioButtonGroup>
  <LanguagesMenuItem>
  <LanguagesMenuItem>
  ...
</KRadioButtonGroup>
```

Reply:
@MisRob Rendering of `LanguagesMenuItem` is being handled through `video.js` menu system. So I wrapped the child components of the menu in `LanguagesMenu` component to wrap the `LanguagesMenuItem` containing `KRadioButton`. 

Also all the requested changes have been completed. Now this should be ready for QA by @radinamatic . 
Looking forward to your feedback !